src/auto-evo/steps/MigrateSpecies.cs

```diff
@@ -99,7 +97,9 @@ public bool RunStep(RunResults results)
                 var moveAmount = (long)random.Next(population * Constants.AUTO_EVO_MINIMUM_MOVE_POPULATION_FRACTION,
                     population * Constants.AUTO_EVO_MAXIMUM_MOVE_POPULATION_FRACTION);
 
-                if (moveAmount > 0 && targetMiche.InsertSpecies(species, patch, null, cache, true, speciesWorkMemory))
+                var insertWorkMemory = new Miche.InsertWorkingMemory(targetMiche);
```

Comment:
Why was this made a local variable? I'd prefer the field approach as it ensures the list isn't recreated each time in this loop which burns through a bunch of unnecessary memory allocations.